package-lock.json

```diff
@@ -948,7 +948,7 @@
       }
     },
     "@nypl/dgx-header-component": {
-      "version": "git+https://git@github.com/NYPL/dgx-header-component.git#3650130f8845f61812dac434a40b85d63020f5b1",
+      "version": "git+https://git@github.com/NYPL/dgx-header-component.git#2913dd935cef141f6176d40426bd8e3a330dcd40",
```

Comment:
I think these changes might not be necessary to commit. The time it did, it came with an update to design-toolkit and to node-sass.